changelog/@unreleased/pr-306.v2.yml

```diff
@@ -0,0 +1,8 @@
+type: improvement
+improvement:
+  description: |-
+    Add generated CLI support for:
```

Comment:
Rather than state what was done, can you re-word this to convey what has changed that an end user would care about if they read the release notes.

For example, instead of just saying "Add generated CLI support for ...", you can say that binary args can now be provided to generated CLIs via a file (using the @ notation), stdin (using the "@-" notation), etc.